portfolio/src/main/webapp/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014-2018 Materialize
```

Comment:
This is the copyright license of the framework I used

Reply:
It is good that you include a copyright license file in your project. I probably will create a folder "license" and put all license files there. You can replace the file name with the framework's name. So it is clear that this license file is related to that framework. Sometimes when you release your product, you might need to merge all license files into a big license file. We have some internal tools to do that. But we can just use the license folder for this project.

Reply:
Thank you for the recommendation. I made a folder and moved it there. There is also a license at the top of the java file and script.js file that came with the starter code. Should I move that into a file and place that under the license folder as well? I think I should, but I don't know if there is a difference when it is at the top of the file verses in a separate file.

Reply:
For the licenses at the top of the java file and script.js file, we should just leave them there. So people knows that this file is protected by a license which usually doesn't contain all information (e.g. distribute/modify rights). When the people needs more information, they can check the detailed information in the license folder.

Reply:
Okay. I understand the difference. Thank you!